messaging/src/test/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessorTest.java

```diff
@@ -958,14 +962,6 @@ private void mockEventHandlerInvoker() {
         );
     }
 
-    @Test
-    void buildWithNullSpanFactoryThrowsAxonConfigurationException() {
-        PooledStreamingEventProcessor.Builder builderTestSubject = PooledStreamingEventProcessor.builder();
```

Comment:
I'm uncertain how this test lost it's value tbh. How "nitty" the test may be.

Reply:
The cast just didn't work, that was the problem (because null on the overload made the compiler confused). 

![image](https://github.com/AxonFramework/AxonFramework/assets/3278878/53158eb5-af60-4ba5-aed8-90869fbaeb85)

Just remembered I can cast null to something though 😂  ` builderTestSubject.spanFactory((SpanFactory) null)`

Reply:
Long story short, the test is back